Refactor iOS documentation (#67)
rootstrap/FlowForms: docs/pages/ios-quickstart.md

Reviewer: At the end of the file, it may be good to have a mention indicating that for more complex use cases the users can refer to the iOS example folder, and a link to the folder in Github, what do you think?

Continuing from the [KMP quickStart](kmp-quickstart)

**2 :** Create an instance of **SignupViewModel** within our **FormState** or class that stores the state of the form:

<pre><code class="swift">
import shared
import SwiftUI

final class FormState: ObservableObject {
  let viewModel = SignupViewModel()

  init() { }
}
</code></pre>

> <p class="comment"> SignupViewModel is located within the common module and its pure kotlin code so it can be used by both platforms</p>

Reviewer: ```suggestion
<p class="comment">Since SignupViewModel is located within the common module and it is pure kotlin code we can use it directly in iOS</p>
```

	
    var body: some View {
        VStack {
            TextField(
                "Email textfield", 
                text: Binding(
                    get: { formState.uiState?.email ?? "" },
                    set: { formState.viewModel.onEmailChange(value: $0) }
                )
            )
            if let emailErrorMessage = formState.uiState?.emailError {
                ErrorMessageView(message: emailErrorMessage)
            }

            SecureField(
                "Password textfield", 
                text: Binding(
                    get: { formState.uiState?.password ?? "" },
                    set: { formState.viewModel.onPasswordChange(value: $0) }
                )
            )
            if let passwordErrorMessage = formState.uiState?.passwordError {
                ErrorMessageView(message: passwordErrorMessage)
            }
            ...
            Button {

>                 // Sign up

Reviewer: Is this comment needed? I think the Text already explains the intention

                    set: { formState.viewModel.onEmailChange(value: $0) }
                )
            )
            if let emailErrorMessage = formState.uiState?.emailError {
                ErrorMessageView(message: emailErrorMessage)
            }

            SecureField(
                "Password textfield", 
                text: Binding(
                    get: { formState.uiState?.password ?? "" },
                    set: { formState.viewModel.onPasswordChange(value: $0) }
                )
            )
            if let passwordErrorMessage = formState.uiState?.passwordError {
                ErrorMessageView(message: passwordErrorMessage)
            }
            ...
            Button {
                signUp()
            } label: {
                Text("Sign Up")
                    .foregroundColor(.black)
            }.disabled(formState.uiState?.isFormValid == false)
        }
    }
}
</code></pre>

And that's one of the easiest forms we can create using **FlowForms**, we don't need to care about managing the field's validation triggering or making any complex logic to enable or disable the signup button.

> For more complex use cases and detailed insights into advanced functionalities and showcases of the **FlowForms** library on iOS, you can refer to the [iOS example app](https://github.com/rootstrap/FlowForms/tree/main/ExampleApp%20iOS) in the official repository. This comprehensive example provides

Reviewer: The ending `This comprehensive example provides` feels like it should say something after `provides` 😅 Could you check?
